core/src/callback.jl

```diff
@@ -379,10 +425,26 @@ end
 
 "Copy the current flow to the SavedValues"
 function save_flow(u, t, integrator)
-    vcat(
-        get_tmp(integrator.p.graph[].flow_vertical, 0.0),
-        get_tmp(integrator.p.graph[].flow, 0.0),
-    )
+    (; tprev, p) = integrator
```

Comment:
There is actually also `dt` in `integrator`, does that help the edge cases a bit?